contracts/MechMarketplace.sol

```diff
@@ -299,7 +299,7 @@ contract MechMarketplace is IErrorsMarketplace {
         bytes32 paymentType,
         address priorityMech,
         uint256 responseTimeout,
-        bytes calldata paymentData
+        bytes memory paymentData
```

Comment:
This change is needed for addressing the "stack too deep" error otherwise. I don't see an issue here as the function is internal.